### DISCUSSION TITLE
Different results between Pyrosim and FDS

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#12918] [Q&A]
I exported a file from Pyrosim to run it using FDS but I noticed that the results are way different!

The simulation is 1X1heptane pool fire and my target to reach a 2.4Mw her.

1x1 hetpane pool fire.fds
Generated by PyroSim - Version 2019.1.0515
May 7, 2024 6:07:43 PM

&HEAD CHID='1x1_hetpane_pool_fire'/
&TIME T_END=100.0/
&DUMP RENDER_FILE='1x1_hetpane_pool_fire.ge1', COLUMN_DUMP_LIMIT=.TRUE., DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh03-a-merged-merged', IJK=15,15,39, XB=3.5,5.5,4.5,6.5,-0.1,5.0/

&REAC ID='HEPTANE',
      FYI='NIST NRC FDS5 Validation',
      FUEL='N-HEPTANE',
      AUTO_IGNITION_TEMPERATURE=0.0,
      CO_YIELD=6.0E-3,
      SOOT_YIELD=0.015,
      HEAT_OF_COMBUSTION=4.8E4,
      RADIATIVE_FRACTION=0.35/

&MATL ID='steel',
      SPECIFIC_HEAT=0.46,
      CONDUCTIVITY=45.8,
      DENSITY=7850.0/
&MATL ID='heptane',
      SPECIFIC_HEAT=2.24,
      CONDUCTIVITY=0.14,
      DENSITY=675.0,
      ABSORPTION_COEFFICIENT=187.5,
      EMISSIVITY=1.0,
      HEAT_OF_REACTION=317.0,
      SPEC_ID(1,1)='N-HEPTANE',
      NU_SPEC(1,1)=1.0,
      BOILING_TEMPERATURE=98.35/

&SURF ID='steel',
      COLOR='BLACK',
      BACKING='VOID',
      MATL_ID(1,1)='steel',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=1.73E-3/
&SURF ID='heptane',
      COLOR='RED',
      BACKING='VOID',
      MATL_ID(1,1)='heptane',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.02/

&OBST ID='Obstruction', XB=3.5,4.033333,5.033333,5.966667,-0.1,0.030769, SURF_ID='INERT'/ 
&OBST ID='Obstruction', XB=3.5,5.5,4.5,5.033333,-0.1,0.030769, SURF_ID='INERT'/ 
&OBST ID='Obstruction', XB=3.5,5.5,5.966667,6.5,-0.1,0.030769, SURF_ID='INERT'/ 
&OBST ID='Obstruction', XB=4.033333,4.966667,5.033333,5.966667,-0.1,0.030769, SURF_IDS='heptane','INERT','INERT'/ 
&OBST ID='Obstruction', XB=4.966667,5.5,5.033333,5.966667,-0.1,0.030769, SURF_ID='INERT'/ 
&OBST ID='Obstruction', XB=4.033333,4.966667,5.033333,5.033333,0.030769,0.161538, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.033333,4.966667,5.966667,5.966667,0.030769,0.161538, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.033333,4.033333,5.033333,5.966667,0.030769,0.161538, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.966667,4.966667,5.033333,5.966667,0.030769,0.161538, SURF_ID='steel'/ 

&VENT ID='Mesh Vent: Mesh03-a-merged-merged [XMAX]', SURF_ID='OPEN', XB=5.5,5.5,4.5,6.5,-0.1,5.0/ 
&VENT ID='Mesh Vent: Mesh03-a-merged-merged [XMIN]', SURF_ID='OPEN', XB=3.5,3.5,4.5,6.5,-0.1,5.0/ 
&VENT ID='Mesh Vent: Mesh03-a-merged-merged [YMAX]', SURF_ID='OPEN', XB=3.5,5.5,6.5,6.5,-0.1,5.0/ 
&VENT ID='Mesh Vent: Mesh03-a-merged-merged [YMIN]', SURF_ID='OPEN', XB=3.5,5.5,4.5,4.5,-0.1,5.0/ 
&VENT ID='Mesh Vent: Mesh03-a-merged-merged [ZMAX]', SURF_ID='OPEN', XB=3.5,5.5,4.5,6.5,5.0,5.0/ 

&TAIL /

**Comment by drjfloyd** [comment]
Pyrosim 2019 is running a 2019 version of FDS vs. whatever version you last installed directly on your PC.

**Reply by HunterXHunter1997** [reply]
can I download an old version from FDS, and if so, can you please guide me?

**Reply by TristanHehnen** [reply]
FDS installer of previous versions are available from the [main FDS GitHub page](https://github.com/firemodels/fds), on the right hand side under ["Releases"](https://github.com/firemodels/fds/releases).

**Reply by HunterXHunter1997** [reply]
I downloaded fds6.7.7 which is compatible with Pyrosim 2019 but also I am having the same issue.
I also tried to re-import the file to Pyrosim but I have the same results

here is a screenshot for the hrr in both cases:

**Reply by HunterXHunter1997** [reply]
[
![Screenshot 2024-05-13 225728](https://github.com/firemodels/fds/assets/168332950/b78c5806-856b-4c1e-bd7d-30c03df6b078)
](url)

**Reply by drjfloyd** [reply]
FDS 6.7.7 was released in 2021 which is two years later than the version of Pyrosim (2019) you are running.  Look at the .out file for the case you ran with Pyrosim. It will tell you what version of FDS was run.  It is probably 6.7.0 or 6.7.1.

You are trying to predict the burning rate of a liquid pool.  This is research use of FDS and an area of the code which is under active development.  I would not run older versions of FDS and would use the latest version of FDS. For this kind of work I would not compare two versions of FDS but rather consider if the current release gives reasonable behavior for your input based on your best knowledge of fire dynamics or the best available data from experiments similar to your simulation.  

 If you have issues running old versions of FDS, we cannot help you. We can only fix demonstrable issues that exist in the current source code.
